seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/TransformConfigConstants.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.common.constants;
+
+public final class TransformConfigConstants {
+    /* common transform constant */
+    public static final String FIELDS = "fields";
+    public static final String SOURCE_FILED = "source_field";
+    public static final String DEFAULT_SOURCE_FILED = "raw_message";
+    public static final String TARGET_FILED = "target_field";
+
+    /* json transform constant */
+    public static final String SCHEMA_DIR = "schema_dir";
+    public static final String SCHEMA_FILE = "schema_file";
+    public static final String DEFAULT_SCHEMA_FILE = "";
+
+    /* replace transform constant */
+    public static final String PATTERN = "pattern";
+    public static final String REPLACEMENT = "replacement";
+    public static final String REPLACE_REGEX = "is_regex";
+    public static final boolean DEFAULT_REPLACE_REGEX = false;
+    public static final String REPLACE_FIRST = "replace_first";
+    public static final boolean DEFAULT_REPLACE_FIRST = false;
+
+    /* split transform constant */
+    public static final String SPLIT_SEPARATOR = "separator";
+    public static final String DEFAULT_SPLIT_SEPARATOR = " ";
+
+    /* UUID transform constant */
+    public static final String UUID_PREFIX = "prefix";
+    public static final String DEFAULT_UUID_PREFIX = "";
+    public static final String UUID_SECURE = "secure";
+    public static final boolean DEFAULT_UUID_SECURE = false;
```

Comment:
The constants file need to put into plugin module, you need to create constant file for each plugin.